tests/writeConcern/writeconcern-serialization-001.phpt

```diff
@@ -0,0 +1,192 @@
+--TEST--
+MongoDB\Driver\WriteConcern serialization
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$tests = [
+    new MongoDB\Driver\WriteConcern(-3), // MONGOC_WRITE_CONCERN_W_MAJORITY
+    new MongoDB\Driver\WriteConcern(-2), // MONGOC_WRITE_CONCERN_W_DEFAULT
+    new MongoDB\Driver\WriteConcern(-1),
+    new MongoDB\Driver\WriteConcern(0),
+    new MongoDB\Driver\WriteConcern(1),
+    new MongoDB\Driver\WriteConcern(MongoDB\Driver\WriteConcern::MAJORITY),
+    new MongoDB\Driver\WriteConcern('tag'),
+    new MongoDB\Driver\WriteConcern(1, 0),
+    new MongoDB\Driver\WriteConcern(1, 0, false),
+    new MongoDB\Driver\WriteConcern(1, 1000),
+    new MongoDB\Driver\WriteConcern(1, 1000, true),
+    new MongoDB\Driver\WriteConcern(-2, 0, true),
+    // Note: wtimeout is only applicable for w > 1
```

Comment:
I believe this was copied (and corrected) from the following tests, which all state "Note: wtimeout is only applicable applies for w > 1".

 * writeconcern-bsonserialize-001.phpt
 * writeconcern-bsonserialize-002.phpt
 * writeconcern-debug-003.phpt
 * writeconcern-var_export-001.phpt

Would you mind correcting those in a separate commit in this PR?